silnlp/alignment/fast_align.py

```diff
@@ -85,9 +85,8 @@ def train(self, src_file_path: Path, trg_file_path: Path) -> None:
         self.model_dir.mkdir(exist_ok=True)
         align_input_path = self.model_dir / "align-input.txt"
 
-        with open(src_file_path, "r", encoding="utf-8") as src_tok_output_file, open(
-            trg_file_path, "r", encoding="utf-8"
-        ) as trg_tok_output_file, open(align_input_path, "w", encoding="utf-8", newline="\n") as align_input_file:
+        with src_file_path.open("r", encoding="utf-8") as src_tok_output_file, trg_file_path."r".open( encoding="utf-8"
```

Comment:
I think `trg_file_path."r".open(` is a typo.

Reply:
Fixed.  Nice catch.